frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/BuilderFinallyNode.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { chart_color_blue_300 as blueColor } from '@patternfly/react-tokens';
+import { observer, Node } from '@patternfly/react-topology';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import { PipelineVisualizationTask } from '../detail-page-tabs/pipeline-details/PipelineVisualizationTask';
+import {
+  BUILDER_NODE_ADD_RADIUS,
+  NODE_WIDTH,
+  NODE_HEIGHT,
+  FINALLY_NODE_PADDING,
+  FINALLY_NODE_VERTICAL_SPACING,
+  FINALLY_ADD_LINK_TEXT_HEIGHT,
+  BUILDER_NODE_ERROR_RADIUS,
+  FINALLY_ADD_LINK_SIZE,
+} from './const';
+import PlusNodeDecorator from './PlusNodeDecorator';
+import ErrorNodeDecorator from './ErrorNodeDecorator';
+import TaskList from './TaskList';
+import { getFinallyTaskHeight } from './utils';
+
+const BuilderFinallyNode: React.FC<{ element: Node }> = ({ element }) => {
```

Comment:
Best type Element better... ie [BuilderNode](https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/BuilderNode.tsx#L17) has the data typed.

You'll need to expand your types to have both expected props for the `FinallyNode` as well as the `BuilderFinallyNode`.